### DISCUSSION TITLE
Make counters' rolling duration adjustable

### DISCUSSION THREAD

**Discussion by leduyquang753** [ppy/osu#13124] [Ideas]
**Describe the new feature:** Currently the counters' display (score, accuracy, combo) have a rolling effect which does not reflect the current stats immediately. I think there should be options to toggle the effect (or at least adjustable).

**Proposal designs of the feature:** There will be three sliders in *Gameplay* → *General*: *Score counter rolling duration*; *Accuracy display rolling duration* and *Combo counter rolling duration*. Alternatively, if simplicity is leaned towards, there will be three toggle options to enable the rolling effect for each counter, or just one to toggle them altogether.

**Comment by peppy** [comment]
This would be a skin decision if anything. I'm not sure we will add this because the rolling is part of what I would consider a game being a game, and not a number simulator.
